classes/tests.rb

```diff
@@ -0,0 +1,10 @@
+require './person'
+require './capitalize_decorator'
+require './trimmer_decorator'
```

Comment:
- [ ] Well done for adding a file that helps us easily see if your code is working 🙏 However, kindly replace `require` with `require_relative` so that your code runs smoothly because it causes an error currently as shown below 👇 
![image](https://github.com/sameedmohsin28/OOP-school-library/assets/46839548/896596e8-8790-4d1b-b651-ddd4c8e21b41)